doc/glenn_utilities/bagging_node.md

```diff
@@ -18,6 +18,6 @@ In `bagging_node.launch`, `bag_topics.yaml` is loaded.
 
 ## Implementation Details
 
-Uses python `subprocess` module to call `rosbag record`. Bags are saved to `$HOME/glenn_bags`.  
+Uses python `subprocess` module to call `rosbag record`. Bags are saved with respect to `ROS_LOG_DIR`. glenn_bags will be parallel to the log folder of ros.  
```

Comment:
What does "glenn_bags will be parallel to the log folder of ros" mean?

Reply:
It means /log and /glenn_bags share the same parent directory.